recipes/libdrs_c/build.sh

```diff
@@ -0,0 +1,17 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
```

Comment:
Try to preserve to the original flags with `LDFLAGS="-L${PREFIX}/lib $LDFLAGS"`.